picom.sample.conf

```diff
@@ -61,6 +61,12 @@ shadow-exclude = [
 # Crop shadow of a window fully on a particular Xinerama screen to the screen.
 # xinerama-shadow-crop = false
 
+# Try to detect windows with rounded corners and don't consider them 
+# shaped windows. The accuracy is not very high, unfortunately.
+#
+# detect-rounded-corners = false
+detect-rounded-corners = true;
```

Comment:
rounded_corners isn't shadow specific, it can be matched against in any type of rules.